### PR TITLE
Validate contactform

### PR DESCRIPTION
**Pull request by HenaryKhuraijam**
### Create Function 
- When the user submits the form, you check if the email is in lower case.
- If the validation is OK, the form is sent. 
- If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.

### CODE REVIEW

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @HenaryKhuraijam, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that 

- [ ] has linter errors. Kindly fix it

linter-error


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by HenaryKhuraijam** [comment]
## Make changes as suggested

File: index.js [hunk 1]
```diff
@@ -11,9 +11,13 @@ function menuClickedHandler() {
...
 document.getElementById('menu').addEventListener('click', menuClickedHandler);
 
-function btn() {
-  const cardone = document.getElementById('cardwork1');
-  cardone.classList.toggle('popup-windows');
-}
-
-document.getElementById('btn1').addEventListener('click', btn);
+document.getElementById('contact-form').addEventListener('submit', (e) => {
+  const errorMsg = document.getElementById('error-message');
```
Comment (Kossi-stack) [inline index.js:15]: - [ ] Kindly fix the error, whenever i submit the email field with uppercase i don't get an error message. I even clone the project locally and run same error.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @HenaryKhuraijam,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.js (hunk 1 above)
Comment (HenaryKhuraijam) [reply index.js:15]: Thank you so much for finding out my mistakes

**Comment by HenaryKhuraijam** [comment]
## Makes changes as suggested

**Review by sumairq — APPROVED** [review]
### Hi @HenaryKhuraijam , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**